manager/controllers/utils/utils.go

```diff
@@ -153,3 +153,15 @@ func Intersection(set1 []string, set2 []string) []string {
 	}
 	return res
 }
+
+const FYBRIKAPPGUID = "FybrikApplicationGUID"
```

Comment:
should we replace GUID to UUID ? 
GUID is more Microsoft style, golnag usually uses UUID.

Reply:
sure